src/pages/AppList.svelte

```diff
@@ -19,44 +23,159 @@
     filteredApps = apps.filter((appItem) => appItem.name.includes(filterText));
   }
   let showDeprecated = false;
+
+  const appLogos = {
+    browser: "/img/app-logos/browser.png",
+    beta: "/img/app-logos/beta.png",
+    amazon: "/img/app-logos/amazon.png",
+    "klar-focus": "/img/app-logos/klar-focus.png",
+    lockwise: "/img/app-logos/lockwise.png",
+    nightly: "/img/app-logos/nightly.png",
+    reality: "/img/app-logos/reality.png",
+    others: "/img/app-logos/mozilla.png",
+    dev: "/img/app-logos/dev.png",
+  };
+
+  function getAppLogo(app) {
+    if (app.match(/fenix/) || app.match(/nightly/)) {
+      return appLogos.nightly;
+    }
+    if (app.match(/beta/)) {
+      return appLogos.beta;
+    }
+    if (app.match(/echo/) || app.match(/fire-tv/)) {
+      return appLogos.amazon;
+    }
+    if (app.match(/focus/) || app.match(/klar/)) {
+      return appLogos["klar-focus"];
+    }
+    if (app.match(/lockwise/)) {
+      return appLogos.lockwise;
+    }
+    if (app.match(/reality/)) {
+      return appLogos.reality;
+    }
+    if (app.match(/dev/)) {
+      return appLogos.dev;
+    }
+    if (app.match(/firefox/)) {
+      return appLogos.browser;
+    }
+    return appLogos.others;
+  }
+
+  function isPlatform(app) {
+    if (
+      includes(app, "iOS") ||
+      includes(app, "Android") ||
+      includes(app, "Amazon")
+    ) {
+      return true;
+    }
+    return false;
+  }
+
+  function getPlatformLogo(app) {
+    if (includes(app, "iOS")) {
+      return "https://i.imgur.com/swlzpbM.jpg";
+    }
+    if (includes(app, "Android")) {
+      return "https://i.imgur.com/ji8NP0b.png";
+    }
+    if (includes(app, "Amazon")) {
+      return "https://i.imgur.com/EwaktM3.png";
+    }
+    return null;
+  }
 </script>
 
 <style>
-  .app-detail {
-    @include text-title-3xs;
-    a {
-      text-decoration: none;
+  .app-filter {
+    margin: $spacing-md $spacing-xl;
+    .deprecation-checkbox {
+      text-align: right;
     }
-    i {
-      @include text-body-md;
+  }
+
+  .app-list {
+    display: flex;
```

Comment:
I think we should use [CSS grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout) for this, it's kind of exactly what it's designed for (flex box is a little more low-level).

A few extra points on what this should look like:

* I think we should try to have more padding between the cards (at least 24px, I think -- don't know what the equivalent would be in protocol tokens)
* I think the model of having a variable number of columns depending on display size is the right one, but I think extra elements (if there is a trailing row) should be *left* aligned, rather than centred (I think you should be able to do this with [grid-auto-flow](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-flow).